build/networking/README_DOCKER.md

```diff
@@ -139,50 +139,55 @@ configuration file settings and inner workings!
 If above commands are successful, at this point you should have your IPDK
 container up and running, and should see the container prompt like below:
 
-```text
-    root@c5efb1a949ad ~]#
+```bash
+  root@c5efb1a949ad ~]#
 ```
 
 `c5efb1a949ad` is your container id as displayed in command `docker ps -a`
 
-
 As your Infrap4d process should be up and running, you could see the infrap4d
 process with the `ps -ef | grep infrap4d` command.
 
 ## Section 2: Running example use case
 
-### Section 2.1: Running example setup with TAP ports.
+### Section 2.1: Running example setup with TAP ports
+
 Infrap4d process is already started with `ipdk start -d` or with manual
 steps mentioned above.
 Start example usecase by connecting to docker and running script `/root/scripts/rundemo_TAP_IO.sh`,
 this script will:
-  - Creates TAP ports and moves to two different namespaces.
-  - Create a forwarding pipeline binary.
-  - Load the target with a forwarding pipeline.
-  - Configure rules.
-  - Test traffic with Ping between TAP ports.
 
+* Creates TAP ports and moves to two different namespaces
+* Create a forwarding pipeline binary
+* Load the target with a forwarding pipeline
+* Configure rules
+* Test traffic with Ping between TAP ports
+
+To cleanup, use command `ipdk rm` to delete existing IPDK docker container
 
 ### Section 2.2: Running example setup ( VM1 <-> IPDK Container <-> VM2 )
+
 Below commands will help you setup traffic between 2 VMs on your host with the
 IPDK container as a P4 program enabled vswitch switching traffic between them.
 
 Pre-requisite:
-- Container in Section 1 should be up and with Infrap4d running
-- If you are running in a VM then make sure you have nested virtualization
+
+* Container in Section 1 should be up and with Infrap4d running
+* If you are running in a VM then make sure you have nested virtualization
   enabled on your guest VM
-- QEMU, KVM, libvirt and cloud-utils packages should be installed as per your distribution and running.
-- `ipdk` commands executed on host machine works only when container is started
+* QEMU, KVM, libvirt and cloud-utils packages should be installed as per your distribution and running.
+* `ipdk` commands executed on host machine works only when container is started
```

Comment:
_work_, not _works_

_on host machine_ is a prepositional phrase. The verb _work_ needs to agree in number with the subject _commands_. 

_ipdk commands ... work_

Reply:
Fixed